src/main/java/FigureInterface.java

```diff
@@ -0,0 +1,5 @@
+public interface FigureInterface {
+    abstract void draw();
+
+    public abstract double areaValue();
+}
```

Comment:
checklist: "Don’t put all behavior into a single interface if the methods are conceptually different from each other."